functions/python-tem-smtp-server/README.md

```diff
@@ -0,0 +1,59 @@
+# Send TEM from a serverless function
+
+This example demonstrates how to send TEM with an SMTP server from Python functions.
+
+It assumes that you are familiar with how Serverless Functions and Scaleway Transactional Email work. 
+If needed, you can check the Scaleway official documentation for serverless functions [here](https://www.scaleway.com/en/docs/serverless/functions/quickstart/) 
+and for TEM [here](https://www.scaleway.com/en/docs/managed-services/transactional-email/quickstart/).
+
+## Requirements
+
+* You have configured your domain with Transactional Email
+* You have generated an API key with the permission `TransactionalEmailFullAccess`
+* You have [Python](https://www.python.org/) installed on your machine
+* You have [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs) installed on your machine
+
+## Testing with serverless offline for Python
+
+In order to test your function locally before the deployment, you can install our offline testing library with:
+
+```bash
+pip install scaleway_functions_python==0.2.0
+```
+
+Export your environment variables and then launch your function locally:
+```bash
+export TEM_PROJECT_ID=<the Project ID of the Project in which the TEM domain was created>
+export SECRET_KEY=<the secret key of the API key of the project used to manage your TEM domain>
+
+python handler.py
+```
+
+Test your local function using `curl`:
+
+```bash
+curl http://localhost:8080
+```
+
+This should email the recipient defined in `handler.py`.
```

Comment:
**issue(flow):** can we make the recipient an environment variable, or even an argument to the script, e.g. `python handler.py foo@bar.com`?

Alternatively we could just add a note saying "Edit handler.py to set the recipient for your mail".

Reply:
I went with the alternative to add a note to edit the file handler.py, because there are multiple variables to configure (recipient, sender and potentially host and port). This is now explained in the "Setup section".